deep-map/index.js

```diff
@@ -5,10 +5,11 @@ export { getPath, setPath } from './path.js'
 
 export function deepMap(initial = {}) {
   let $deepMap = atom(initial)
-  $deepMap.setKey = (key, value) => {
-    if (getPath($deepMap.value, key) !== value) {
-      $deepMap.value = setPath($deepMap.value, key, value)
-      $deepMap.notify(key)
+  $deepMap.setKey = (key, newVal) => {
+    let oldVal = getPath($deepMap.value, key)
+    if (oldVal !== newVal) {
```

Comment:
Could you use `newValue` and `oldValue` name? `val` looks like to cryptic for me.